libs/dh/admin/data-access-api/src/lib/dh-admin-user-management-data-access-api.store.ts

```diff
@@ -81,12 +95,20 @@ export class DhAdminUserManagementDataAccessApiStore extends ComponentStore<DhUs
           .pipe(
             tapResponse(
               (response) => {
-                this.setLoading(LoadingState.LOADED);
+                this.setInitialPageLoaded(state);
+
+                this.patchState({
```

Comment:
`this.patchState(..` can be moved to `this.setInitialPage..` and then removed on line 109 to avoid duplication.

Reply:
I have some changes locally which remove a couple of properties. Will push them soon.